### ISSUE TITLE
Auto-class decoration algorithm improvements

### ISSUE THREAD

**Issue by oleg-shilo** [oleg-shilo/cs-script#37]
Auto-class decoration should allow conditional injection of `using static dbg;` for transpaent use of Python-like `print`:
```C#
//css_args -ac
using System;
using System.IO;

void main()
{
    print(Directory.GetFiles(".", "*"));
}
```
----

The algorithm also need to detect class-full scripts and disable the decoration. This in urn will allow having the auto-class decoration enabled system wide (with `-ac` switch).

**Comment by oleg-shilo** [comment]
Done with 11d5c61fffa2e0c561c7ba02e7616bbc02bbdd7a
Planned for v3.21 release

**Comment by oleg-shilo** [comment]
Fixed in Release v3.21.1
